### PR TITLE
ddl: Return an error when add or drop primary key

### PR DESCRIPTION
**Pull request by zimulala**
Fix issue #2202

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @coocood @XuHuaiyu @tiancaiamao

**Comment by coocood** [comment]
LGTM

File: ddl/ddl.go [hunk 1]
```diff
@@ -82,6 +82,9 @@ var (
 	ErrInvalidIndexState = terror.ClassDDL.New(codeInvalidIndexState, "invalid index state")
 	// ErrInvalidForeignKeyState returns for invalid foreign key state.
 	ErrInvalidForeignKeyState = terror.ClassDDL.New(codeInvalidForeignKeyState, "invalid foreign key state")
+	// ErrUnsupportedModifyPrimaryKey returns an error when add or drop the primary key.
+	// It's exproted for testing.
```
Comment (XuHuaiyu) [inline ddl/ddl.go]: s/exproted/exported?

Comment (XuHuaiyu) [inline ddl/ddl.go]: never mind

**Pull request merged**